server/src/main/java/com/google/android/attestation/AuthorizationList.java

```diff
@@ -771,20 +771,17 @@ private static ParsedAuthorizationMap getAuthorizationMap(ASN1Encodable[] author
     return new ParsedAuthorizationMap(authorizationMap, ImmutableList.copyOf(unorderedTags));
   }
 
-  private static LocalDate toLocalDate(String value) {
+  @VisibleForTesting
+  static LocalDate toLocalDate(String value) {
```

Comment:
Nit: Please note that this is not intended to be a permanent work around somewhere in here.